src/video/emscripten/SDL_emscriptenvideo.c

```diff
@@ -297,6 +297,7 @@ static int Emscripten_SetWindowFullscreen(SDL_VideoDevice *_this, SDL_Window *wi
 
         if (fullscreen) {
             EmscriptenFullscreenStrategy strategy;
+            memset(&strategy, 0, sizeof(strategy));
             SDL_bool is_fullscreen_desktop = !window->fullscreen_exclusive;
 
             strategy.scaleMode = is_fullscreen_desktop ? EMSCRIPTEN_FULLSCREEN_SCALE_STRETCH : EMSCRIPTEN_FULLSCREEN_SCALE_ASPECT;
```

Comment:
Thanks! FYI, the typical SDL coding pattern would be:
```c
        if (fullscreen) {
            EmscriptenFullscreenStrategy strategy;
            SDL_bool is_fullscreen_desktop = !window->fullscreen_exclusive;

            SDL_zero(strategy);
            strategy.scaleMode = is_fullscreen_desktop ? EMSCRIPTEN_FULLSCREEN_SCALE_STRETCH : EMSCRIPTEN_FULLSCREEN_SCALE_ASPECT;
```
I can make that change, or you can update your PR if you want.

Reply:
Fixed